config/config_json_test.go

```diff
@@ -756,6 +756,8 @@ func TestDiameterAgentJsonCfg(t *testing.T) {
 		Concurrent_requests:  utils.IntPointer(-1),
 		Synced_conn_requests: utils.BoolPointer(false),
 		Asr_template:         utils.StringPointer(""),
+		Rar_template:         utils.StringPointer(""),
+		Disconnect_method:    utils.StringPointer(utils.MetaASR),
```

Comment:
please rename to forced_disconnect and use as possible values: *none(default), *asr, *rar